app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_name_to_copy = command.split(" ")[1]
+    new_file_name = command.split(" ")[2]
+
+    if file_name_to_copy == new_file_name:
+        return
```

Comment:
Mayeby better to use != conditions to avoid return. And also what if command will be not "cp"?